src/components/Card.js

```diff
@@ -73,56 +80,71 @@ export const Card = () => {
+              height: currentSize + 'px',
+              width: currentSize + 'px',
+            }}
             value={qrvalue}
-            size={175}
+            size={currentSize}
           />
 
-          <div className="inputs">
-            <label>{t('wifi.name')}</label>
-            <textarea
-              id="ssid"
-              type="text"
-              maxLength="32"
-              placeholder={t('wifi.name.placeholder')}
-              autoComplete="off"
-              autoCorrect="off"
-              autoCapitalize="none"
-              spellCheck="false"
-              value={network.ssid}
-              onChange={(e) => setNetwork({ ...network, ssid: e.target.value })}
-            />
-            <label
+          <div>
+            <div
               className={`
+                ${currentSize == smallSize ? 'inputs-small' : false}
+                ${currentSize == mediumSize ? 'inputs-medium' : false}
+                ${currentSize == largeSize ? 'inputs-large' : false}
+              `}
+            >
+              <label>{t('wifi.name')}</label>
+              <textarea
+                id="ssid"
+                type="text"
+                maxLength="32"
+                placeholder={t('wifi.name.placeholder')}
+                autoComplete="off"
+                autoCorrect="off"
+                autoCapitalize="none"
+                spellCheck="false"
+                value={network.ssid}
+                onChange={(e) =>
+                  setNetwork({ ...network, ssid: e.target.value })
+                }
+              />
+              <label
+                className={`
                 ${network.hidePassword && 'no-print hidden'}
                 ${network.encryptionMode === 'nopass' && 'hidden'}
               `}
-            >
-              {t('wifi.password')}
-            </label>
-            <textarea
-              id="password"
-              type="text"
-              className={`
+              >
+                {t('wifi.password')}
+              </label>
+              <textarea
+                id="password"
+                type="text"
+                className={`
                 ${network.hidePassword && 'no-print hidden'}
                 ${network.encryptionMode === 'nopass' && 'hidden'}
               `}
-              style={{
-                height:
-                  portrait && network.password.length > 40 ? '5em' : 'auto',
-              }}
-              maxLength="63"
-              placeholder={t('wifi.password.placeholder')}
-              autoComplete="off"
-              autoCorrect="off"
-              autoCapitalize="none"
-              spellCheck="false"
-              value={network.password}
-              onChange={(e) => {
-                setNetwork({ ...network, password: e.target.value });
-              }}
-            />
-
+                style={{
+                  height:
+                    portrait || (!portrait && network.password.length > 40)
```

Comment:
What's this change? Was there a bug here before?

Reply:
Yes, because some letters may be obscured after resizing to medium or large:
![card2](https://user-images.githubusercontent.com/26818304/126959364-9412b3ec-e9d2-4130-ac44-e484fc737a8c.JPG)
![card1](https://user-images.githubusercontent.com/26818304/126959372-2669f188-bf60-4db7-8573-eff22b67981f.JPG)